open-api/rest-catalog-open-api.yaml

```diff
@@ -1453,6 +1456,23 @@ components:
         type: string
       example: "sales"
 
+    data-access:
+      name: X-Iceberg-Data-Access
+      in: header
+      description: >
+        Optional signal to the server that the client supports data access via
+        a comma-separated list of access mechanisms.  The server may choose to 
+        supply access via any or none of the requested mechanisms.
+      required: false
+      schema:
+        type: string
+        enum:
+          - vended_credentials
+          - remote_signing
```

Comment:
Enums prefer kebab-case to snake_case for consistency throughout.

Reply:
done